hammers/jackhammer

```diff
@@ -501,8 +517,20 @@ if __name__ == '__main__':
     dump_parser = subparsers.add_parser('dump', help='Dumps all docker logs')
     dump_parser.add_argument('slots', nargs='*', type=int,
                              help='Specifies the slots to dump rogue states')
+    dump_parser.add_argument('--dump-rogue', action='store_true',
+                             help="If true, will attempt to connect to pysmurf smurf and dump the rogue tree")
     dump_parser.set_defaults(func=dump_func)
 
+    ########### Jackhammer dump parser ###########
+    dump_parser = subparsers.add_parser('deactivate', help='Deactivates slots')
+    dump_parser.add_argument('slots', nargs='*', type=int,
+                             help='Specifies the slots to deactivate')
+    dump_parser.set_defaults(func=deactivate_func)
+
+    dump_parser = subparsers.add_parser('activate', help='Activates slots')
+    dump_parser.add_argument('slots', nargs='*', type=int,
```

Comment:
Should these be nargs='+' , so `slots` is a required argument? As-is, if not provided with any arguments this will drop you into the command line on the shelf manager, which might be confusing